Resources/Prototypes/Datasets/tips.yml

```diff
@@ -123,3 +123,48 @@
+  - "If you accidentally left the game for a long time, you can still check on your character. Perhaps someone shoved them into a cryopod and you can use the 'un-cryo' button?"
+  - "If you see a person who's fallen SSD, make sure to drag them to cryo and shove them into a cryopod. It will prevent them from dying, and will allow the player to return to the body later!"
+  - "Vending machine food is bad for your health! Watch out for food trucks such as the Skipper or the McCargo - they can offer delicious food for a reasonable price!"
+  - "Meals cooked by chefs are more filling, more flavorful, and in some cases can even boost your movement speed for a short time!"
+  - "Each ID card can have a shuttle attached to it. If you need to purchase an additional shuttle, you can ask your local SR for another ID!"
+  - "The Station Representative can rename your shuttle! Usually it's enough to hand your ID and ask them to do it."
+  - "If your ship has a big crew, you can ask the SR for radio keys for your fellow crewmates. Some channels, such as Cargo or Science, are usually empty."
+  - "Before joining an ongoing round, you can see the list of ships registered on the New Frontiers. Perhaps some of them have jobs you would prefer?"
+  - "Don't forget to buy a medical implant before wandering into deep space! It will send a message to the local medics in the event of your death."
+  - "As a medic, you should cooperate with others when there's a medical emergency! It never helps when two medical ships arrive at the same emergency signal."
+  - "As a medic, you can ask the local SR for a medical PDA. It can double as a health analyzer!"
+  - "Medical ships often have access to the hypospray. An experienced chemist can turn it into a lethal weapon, or use it to safely put people to sleep."
+  - "If you're hurt, try to ask for medical help on the radio. Medics are often eager to heal the wounded, as their job usually gets boring after hours of waiting."
+  - "Always carry a GPS with yourself! It can help you locate your station, or tell your location to others and get saved."
+  - "Smuggling is very profitable, but also very risky! Remember to have an IFF console and hide your IFF before going for a dead drop."
+  - "As a smuggler, try to destroy any evidence of your crimes. Remember, your PDA has a notepad app, which can be used to store important notes in a digital form."
+  - "Security has a special IFF console that completely hides their ship on the radar! If you're an outlaw, watch out for those invisible Prowlers!"
+  - "Certain ships can act as motherships. They have shipyard consoles, which allow you to purchase and sell other smaller ships!"
+  - "As a crewmember, you can start a mutiny if your captain refuses to pay you. Be careful though, as the captain has the permission to execute the mutiny leader!"
```

Comment:
1. A lawful mutiny requires the captain to have broken Space Law (non-payment is a civil matter)
2. Non-payment can be resolved through a lawsuit, or security may be willing to intervene
3. An unlawful mutiny would need to first follow escalation guidelines

I think keeping this tip may encourage bad behavior.

Reply:
i feel like, if you reword it to something along the lines of "Check your space law book, found from your PDA, for some useful information on law! For example: The rules for mutiny on the captain if they don't pay you!

Reply:
encourages people not only to see IC law, but also achieves some of the same effect

Reply:
+1 for a tip on finding Space Law in your PDA, but mutiny for non-payment is a crime. There is also nothing explicitly worded around non-payment in Space Law.